cabot/urls.py

```diff
@@ -44,98 +44,62 @@
+    url(r'^result/(?P<pk>\d+)/',               view=StatusCheckResultDetailView.as_view(), name='result'),
+    url(r'^shifts/(?P<pk>\d+)/',               view=ShiftListView.as_view(),               name='shifts-detail'),
+    url(r'^shifts/',                           view=ScheduleListView.as_view(),            name='shifts'),
+    url(r'^graphite/',                         view=graphite_api_data,                     name='graphite-data'),
+    url(r'^user/(?P<pk>\d+)/profile/$',        view=UserProfileUpdateView.as_view(),       name='user-profile'),
+    url(r'^user/(?P<pk>\d+)/profile/(?P<alerttype>.+)', view=UserProfileUpdateAlert.as_view(), name='update-alert-user-data'), # noqa
+    url(r'^schedule/create/',                           view=ScheduleCreateView.as_view(),     name='create-schedule'),
+    url(r'^schedule/update/(?P<pk>\d+)/',               view=ScheduleUpdateView.as_view(),     name='update-schedule'),
+    url(r'^schedule/delete/(?P<pk>\d+)/',               view=ScheduleDeleteView.as_view(),     name='delete-schedule'),
+
+    url(r'^grafana/instance/$',                         view=GrafanaInstanceSelectView.as_view(),                  name='grafana-instance-select'),  # noqa
+    url(r'^grafana/dashboard/$',                        view=GrafanaDashboardSelectView.as_view(),                 name='grafana-dashboard-select'), # noqa
+    url(r'^grafana/panel/$',                            view=GrafanaPanelSelectView.as_view(),                     name='grafana-panel-select'),     # noqa
+    url(r'^grafana/series/$',                           view=GrafanaSeriesSelectView.as_view(),                    name='grafana-series-select'),    # noqa
+    url(r'^grafana/edit/(?P<pk>\d+)/',                  view=GrafanaEditView.as_view(),                            name='grafana-edit'),             # noqa
+    url(r'^grafana/elasticsearch/create/$',             view=GrafanaElasticsearchStatusCheckCreateView.as_view(),  name='grafana-es-create'),        # noqa
+    url(r'^grafana/elasticsearch/update/(?P<pk>\d+)/',  view=GrafanaElasticsearchStatusCheckUpdateView.as_view(),  name='grafana-es-update'),        # noqa
+    url(r'^grafana/elasticsearch/refresh/(?P<pk>\d+)/', view=GrafanaElasticsearchStatusCheckRefreshView.as_view(), name='grafana-es-refresh'),       # noqa
+    url(r'^grafana/refresh/(?P<pk>\d+)/',               view=GrafanaInstanceSelectView.as_view(),                  name='grafana-refresh'),          # noqa
+    url(r'^grafana/dashboard/(?P<pk>\d+)/',             view=GrafanaDashboardSelectView.as_view(),                 name='grafana-dashboard-select'), # noqa
+    url(r'^grafana/panel/(?P<pk>\d+)/',                 view=GrafanaPanelSelectView.as_view(),                     name='grafana-panel-select'),     # noqa
+    url(r'^grafana/series/(?P<pk>\d+)/',                view=GrafanaSeriesSelectView.as_view(),                    name='grafana-series-select'),    # noqa
```

Comment:
not sure about this one. Would still like to follow the #chars per line < 120 rule (since otherwise things are cut off on github and you have to scroll). Also I don't really like lining things up like this because it's annoying to maintain